tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -706,34 +707,31 @@ describe('Filing History List - redirections', () => {
     // find and click the drop-down menu button
     const menuButton = wrapper.find('.menu-btn')
     expect(menuButton).toBeDefined()
-    menuButton.trigger('click')
-    await flushPromises()
+    await menuButton.trigger('click')
 
     // find and click the "File a Correction" menu item
     const fileCorrectionItem = wrapper.find('.file-correction-item')
-    expect(fileCorrectionItem).toBeDefined()
-    fileCorrectionItem.trigger('click')
-    await flushPromises()
+    expect(fileCorrectionItem.exists()).toBe(true)
+    await fileCorrectionItem.trigger('click')
 
     // verify to display the correction dialog modal
-    const fileCorrectionDialog = wrapper.find('#correction-filing-dialog')
-    expect(fileCorrectionDialog).toBeDefined()
```

Comment:
This was incorrect: the wrapper is defined -- it's just an "empty wrapper".

Same below.